### PR TITLE
Update readme and start script

### PR DESCRIPTION
**Pull request by longxice**
- Remove available scripts in the readme
- Change start script: server to backend, start to frontend
- Update high quality gif

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,38 +48,16 @@ REACT_APP_PRIVATE_KEY="your_PRIVATE_KEY_here"
...
-Create a production build
-
-
-```
-npm run build
+npm run backend
 ```
 
-Start the application from the built files
+1. Start the frontend
```
Comment (leonthaibao) [inline README.md]: Should it be step `2` right?

Comment (longxice) [reply README.md]: @leonthaibao fixed

**Comment by anhldbk** [comment]
@longxice lgtm

**Comment by longxice** [comment]
@anhldbk I have just committed codes that remove script 'backend-prod' in the package file

**Pull request merged**